linkerd/app/core/src/serve.rs

```diff
@@ -75,7 +75,8 @@ pub async fn serve<M, S, I, A>(
                 }
             }
         }
-    };
+    }
+    .in_current_span();
```

Comment:
I'm surprised this is necessary but without it we don't see the inbound/outbound span on log messages from this future